app/scripts/modules/ecs/src/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -336,7 +337,7 @@ export class EcsServerGroupConfigurationService {
       .filter({
         account: command.credentials,
         region: command.region,
-        purpose: command.subnetType,
+        purpose: command.subnetTypes ? command.subnetTypes[0] : command.subnetType,
```

Comment:
Users can only deploy to one VPC at a time (multiple subnets that exist in the same VPC). In this case, all of the subnets will be in the same VPC, so it shouldn't matter which subnet you take, since you're using it to get the vpc id.